### ISSUE TITLE
Limit which tags wrap to own lines?

### ISSUE THREAD

**Issue by smlombardi** [welovewordpress/SublimeHtmlTidy#24]
It seems a bit of a waste of space to have some tags like option (in select menus) waste 3 lines for each option item (since the opening tag, value, and closing tag are each put on own line).

Is there and way to specify which tags should do this and which don't?

**Comment by fitnr** [comment]
Setting `"indent":false` in your User HtmlTidy.sublime-settings file will result in far less indentation and fewer lines.

**Comment by smlombardi** [comment]
There's no way to specify which tags?  In BBEdit, there's a Source.profile
file you can edit to specify how to hand each tag, as in break before,
break after and/or break inside.

**Comment by fitnr** [comment]
No, that feature doesn't exist. This package is a wrapper for [libTidy](http://tidy.sourceforge.net/). Adding this feature would require additions to libTidy. You may want to take a look at the [tidy-html5 library](https://github.com/w3c/tidy-html5), which is a fork of libTidy.

**Comment by ricardozea** [comment]
I found a temporary solution/workaround, you can read about it here: http://stackoverflow.com/questions/13143850/sublime-text-2-format-html-tags-content-in-one-line/13145975#13145975

As I said in that post, it's not ideal by any means, but...

Good luck

**Comment by fitnr** [comment]
I didn't know about reindent. It should be possible to call it within the plugin, and remove the need for the workaround.

**Comment by ricardozea** [comment]
Cool. Is this something you would incorporate in the plugin? Or is this something that 'welovewordpress' would do?

Also, why does HTMLTidy remove everything from the HTML file leaving only the markup that's inside the < body > tag?

Thanks.
